src/config.ts

```diff
@@ -380,5 +380,11 @@ export const TEST_LOGIN_PASSWORD = NonEmptyString.decode(
   process.env.TEST_LOGIN_PASSWORD
 );
 
+// When the application is running as Azure Function
+// Express Body parser must be disabled.
+export const DISABLE_BODY_PARSER = fromNullable(process.env.DISABLE_BODY_PARSER)
+  .map(_ => _.toLocaleLowerCase() === "true")
+  .getOrElse(false);
```

Comment:
Imho we can avoid using an env variable and just use true/false whether we instantiate the app in `server.ts` or `StartBackend/index.ts`.

(it's not clear to me if we think on keeping both classic and functions deploy type)

Reply:
Fixed in e641ee7